sdk/storage/azure-storage-blob/azure/storage/blob/_upload_helpers.py

```diff
@@ -254,7 +263,7 @@ def upload_page_blob(
                 kwargs['encryptor'] = encryptor
                 kwargs['padder'] = padder
 
-        kwargs['modified_access_conditions'] = ModifiedAccessConditions(if_match=response['etag'])
+        kwargs['modified_access_conditions'] = ModifiedAccessConditions(if_match=response['etag'])  #type: ignore
```

Comment:
Error here?

Reply:
azure\storage\blob\_upload_helpers.py:266: error: Value of type "None" is not indexable  [index]

Reply:
Cast it